README.md

```diff
@@ -49,6 +49,10 @@ cp okular.sh ~/gnu-koryavov/
 
 Чтобы `gnu-koryavov` запускал Ваш скрипт вместо `okular.sh`, в файле `~/gnu-koryavov/config.conf` измените значение параметра `pdfviewer_script` на путь к вашему скрипту, как это сделано по умолчанию. *Не забудьте поделиться этим скриптом со всеми пользователями `gnu-koryavov`, создав соответсвующий пулл-реквест!*
 
+### Готовые скрипты для просмотра
+- Ocular
```

Comment:
Probably just a typo, switch to `Okular`,

Reply:
Yes, fixed